src/Servers/HttpSys/src/RequestProcessing/Response.cs

```diff
@@ -730,6 +730,12 @@ internal void SendOpaqueUpgrade()
             }
         }
 
+        internal void Transfer()
```

Comment:
```suggestion
        internal void MarkTransfered()
```

Reply:
Ahh! It's `MarkTransferred` with 2 `r`s.